### PR TITLE
iddsampledriver-ge9-np: Add version 0.0.1.2

### PR DESCRIPTION
**Pull request by sitiom**
Windows Indirect Display Driver using the IddCx class extension driver. Virtual monitors for Windows 10/11. @ge9's fork with support for configuring monitor options via `option.txt`

- Depends on nefcon to properly install the driver

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/iddsampledriver-ge9-np.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{
+    "version": "0.0.1.2",
+    "description": "Windows Indirect Display Driver using the IddCx class extension driver. Virtual monitors for Windows 10/11.",
+    "homepage": "https://github.com/ge9/IddSampleDriver",
+    "license": "Unknown",
```
Comment (issaclin32) [inline bucket/iddsampledriver-ge9-np.json]:
```suggestion
    "license": "MIT,CC0 | Public Domain",
```

https://github.com/ge9/IddSampleDriver#readme
```
License MIT and CC0 or Public Domain (for changes I made, check with Microsoft for their license), whichever is least restrictive -- Use it
```

File: bucket/iddsampledriver-ge9-np.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            "Remove-Item \"$dir\\installCert.bat\"",
+            "",
+            "# Install certificates",
+            "certutil -addstore -f root \"$dir\\IddSampleDriver.cer\" | Out-Null",
+            "certutil -addstore -f TrustedPublisher \"$dir\\IddSampleDriver.cer\" | Out-Null",
+            "",
+            "# Copy option.txt to C:\\IddSampleDriver if it does not exist",
+            "if (!(Test-Path \"C:\\IddSampleDriver\\option.txt\")) {",
+            "    Write-Host -Foreground Yellow \"'C:\\IddSampleDriver\\option.txt' not found. Creating...\"",
+            "    Copy-Item -Path \"$dir\\option.txt\" -Destination \"C:\\IddSampleDriver\"",
```
Comment (issaclin32) [inline bucket/iddsampledriver-ge9-np.json]:
```suggestion
            "    ensure 'C:\\IddSampleDriver' | Out-Null",
            "    Copy-Item -Path \"$dir\\option.txt\" -Destination 'C:\\IddSampleDriver\\'",
```
`Copy-Item` will not create the folder for you. Therefore need to create `C:\IddSampleDriver` first.

`ensure` is a Scoop helper function that creates a folder if it does not exist.

File: bucket/iddsampledriver-ge9-np.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
+{
+    "version": "0.0.1.2",
+    "description": "Windows Indirect Display Driver using the IddCx class extension driver. Virtual monitors for Windows 10/11.",
+    "homepage": "https://github.com/ge9/IddSampleDriver",
+    "license": "Unknown",
+    "notes": "Use \"C:\\IddSampleDriver\\option.txt\" to set the number of monitors and resolutions, then restart the driver.",
```
Comment (issaclin32) [inline bucket/iddsampledriver-ge9-np.json]: How do I "restart the driver"? Do you mean re-loading it in *Device Manager*?

File: bucket/iddsampledriver-ge9-np.json (hunk 1 above)
Comment (sitiom) [reply bucket/iddsampledriver-ge9-np.json]: Oh, missed this note. Thanks.

File: bucket/iddsampledriver-ge9-np.json (hunk 3 above)
Comment (sitiom) [reply bucket/iddsampledriver-ge9-np.json]: Yes, or disabling and enabling it via other methods.

Comment (issaclin32) [reply bucket/iddsampledriver-ge9-np.json]: I suggest adding more info like `Then, restart the driver via Device Manager or (some app)`

Comment (sitiom) [reply bucket/iddsampledriver-ge9-np.json]: The Device Manager is a general way to do it, it's up to them now if they use some other app 👍

File: bucket/iddsampledriver-ge9-np.json [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+            "",
+            "# Install driver",
+            "nefconw --create-device-node --hardware-id ROOT\\iddsampledriver --class-name Display --class-guid 4d36e968-e325-11ce-bfc1-08002be10318",
+            "nefconw --install-driver --inf-path \"$dir\\IddSampleDriver.inf\""
+        ]
+    },
+    "uninstaller": {
+        "script": [
+            "if ($cmd -ne 'uninstall') { return }",
+            "nefconw --remove-device-node --hardware-id ROOT\\iddsampledriver --class-guid 4d36e968-e325-11ce-bfc1-08002be10318"
```
Comment (issaclin32) [inline bucket/iddsampledriver-ge9-np.json]:
```suggestion
            "nefconw --remove-device-node --hardware-id ROOT\\iddsampledriver --class-guid 4d36e968-e325-11ce-bfc1-08002be10318",
            "if ($purge) {",
            "    Write-Host -Foreground Yellow \"Removing 'C:\\IddSampleDriver'\"",
            "    Remove-Item 'C:\\IddSampleDriver' -Force -Recurse",
            "}"
```

This allows users to remove config when uninstalling the app via `-purge` or `-p` switch.

In packages using **persist**, `-purge` will remove all the persisted files.

**Comment by issaclin32** [comment]
Tested locally. Works fine.

**Review by issaclin32 — APPROVED** [review]

**Pull request merged**